app.js

```diff
@@ -18,6 +18,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// Always run in UTC, no matter what
+process.env.TZ = 'UTC';
```

Comment:
Is this required? Wouldn't we always be working with UTC timestamp when using date objects and "getTime()"?